docker/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+if [ ! -f /config/qBittorrent/qBittorrent.conf ]; then
+    mkdir -p /config/qBittorrent/
+    cat << EOF > /config/qBittorrent/qBittorrent.conf
+[AutoRun]
+enabled=false
+program=
+[LegalNotice]
+Accepted=false
```

Comment:
Still in effect: https://github.com/qbittorrent/qBittorrent/pull/11138#discussion_r318592592

Reply:
that's just the default, as per line 22 trought 26 it checks if the environmental variable "LEGAL" has the value of "accept"
if it has the config is edited to "Accepted=true" otherwise to "Accepted=false", inspired by [#11138 (comment)](https://github.com/qbittorrent/qBittorrent/pull/11138#discussion_r619262719)

I'm open to suggestion for changing the varible/default/behaviour, I don't have the skill to touch the webUI and create the appropriate window

Reply:
OK, nice.